cleanreg.py

```diff
@@ -55,6 +56,8 @@ def parse_arguments():
     parser.add_argument('-k', '--keepimages', help="Amount of images (not tags!) which should be kept "
                                                    "for the given repo (if -n is set) or for each repo of the "
                                                    "registry (if -cf is set).", type=int)
+    parser.add_argument('-re', '--regex', help="Image tags matching the regular expression will be kept", default=None)
```

Comment:
see my other comment. I think this flag should only enable regex and not define the tagname. The tagname can be part of of the reponame, e.g `postgres:3.6` where postgres is the image name and 3.6 the tag